README.md

```diff
@@ -66,3 +66,45 @@ This middleware should be declared *after* your other routes but *before* your e
 
 `translate` can be the HOF i18n translate function
 
+## Deep translate
+
+deepTranslate middleware supports nested conditional translations in order to show different content in different scenarios. The middleware adds a `translate` function to `req` which is used in various points throughout the architecture.  This middleware must be applied before any other middleware which rely on the `req.translate` function. Also when initializing the form wizard, or template mixins, if a `translate` function is provided, this will be used rather than the deepTranslate middleware.
+
```

Comment:
@JoeChapman I meant that bit a more of a 'warning' as it will break the middleware - you wouldn't want to override it in that way - its one or the other